vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomActivity.kt

```diff
@@ -69,20 +69,35 @@ class CreateRoomActivity : VectorBaseActivity<ActivitySimpleBinding>(), ToolbarC
                 .subscribe { sharedAction ->
                     when (sharedAction) {
                         is RoomDirectorySharedAction.Back,
-                        is RoomDirectorySharedAction.Close -> finish()
+                        is RoomDirectorySharedAction.Close             -> finish()
```

Comment:
I think it is OK to always pass RESULT_CREATED_ROOM_ID for this two cases.

Reply:
with a null value?

Reply:
Just forget about my remark.